src/app/app.ts

```diff
@@ -84,9 +83,6 @@ class AppImpl implements App {
     createZDSubscription = async (): Promise<AppCallResponse> => {
         // get zendesk client for user
         const zdClient = await newZDClient(this.zdOptions);
-        const req = zdClient.triggers.definitions() || '';
```

Comment:
I assume some of these are removed because they are not used. Is there a lint-like(or other tool) that we can use to detect unused vars?

Reply:
I created an issue to configure the eslint rules to match the mattermost-webapp rules.  https://github.com/mattermost/mattermost-app-zendesk/issues/77